homeassistant/components/venstar/climate.py

```diff
@@ -120,6 +120,8 @@ def __init__(
             HVAC_MODE_COOL: self._client.MODE_COOL,
             HVAC_MODE_AUTO: self._client.MODE_AUTO,
         }
+        self._attr_unique_id = f"{config.entry_id}"
```

Comment:
```suggestion
        self._attr_unique_id = config.entry_id
```

entry_id is already typed to be a string